discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -213,6 +213,21 @@ def wait_for_one_host_to_be_in_wrong_boot_order(self, fall_on_error_status=True)
             fall_on_error_status=fall_on_error_status,
         )
 
+    def wait_for_hosts_to_be_in_wrong_boot_order(
+            self,
+            nodes_count=env_variables['num_nodes'],
+            timeout=consts.CLUSTER_INSTALLATION_TIMEOUT,
+            fall_on_error_status=True
+    ):
+        utils.wait_till_all_hosts_are_in_status(
+            client=self.api_client,
+            cluster_id=self.id,
+            statuses=[consts.ClusterStatus.INSTALLING_PENDING_USER_ACTION],
+            nodes_count=nodes_count,
```

Comment:
nodes_count should have a default of num_nodes

Reply:
it already has..

Reply:
OK, sorry missed that